go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/kubeedge/beehive v0.0.0
 	github.com/kubeedge/viaduct v0.0.0
-	github.com/kubernetes-csi/csi-lib-utils v0.6.1
```

Comment:
Why bump to v0.11.0? Maybe we can update to the latest version.

Reply:
When I upgrade to the newer 0.14 version, it succeeds directly, but when I upgrade to a version after 0.15, I get an error. It looks like I need to manually update a series of dependencies. Do we still need to upgrade it to the latest version? @fisherxu 
![vendor](https://github.com/kubeedge/kubeedge/assets/48212997/a2fa3316-47c8-47b0-9858-772d9f242e35)